### PR TITLE
Adding support px-backup license tests

### PR DESCRIPTION
**Pull request by prashanthpx**
Added the following:
1. License driver interface
2. WaitForLicenseActivation() for inspecting license context after activation.

### CODE REVIEW

File: drivers/backup/portworx/portworx.go [hunk 1]
```diff
@@ -720,6 +722,49 @@ func (p *portworx) WaitForRunning(
...
+
+		// Check if we got response from license server
+		if len(resp.GetResponseInfo()) == 0 {
+			licenseErr = fmt.Errorf("failed to activate license for orgID %v", req.GetOrgId())
+			return nil, false, licenseErr
+		}
+		// iterate over the feature and check if valid feature is present
+		for _, ft := range resp.GetResponseInfo() {
+			if ft.GetName() != "ClusterCount" {
+				licenseErr = fmt.Errorf("found iunvalid feature name")
```
Comment (siva-portworx) [inline drivers/backup/portworx/portworx.go:751]: typo iunvalid

File: drivers/backup/portworx/portworx.go [hunk 2]
```diff
@@ -720,6 +722,49 @@ func (p *portworx) WaitForRunning(
...
+		}
+
+		// Check if we got response from license server
+		if len(resp.GetResponseInfo()) == 0 {
+			licenseErr = fmt.Errorf("failed to activate license for orgID %v", req.GetOrgId())
+			return nil, false, licenseErr
+		}
+		// iterate over the feature and check if valid feature is present
+		for _, ft := range resp.GetResponseInfo() {
+			if ft.GetName() != "ClusterCount" {
```
Comment (siva-portworx) [inline drivers/backup/portworx/portworx.go:750]: It is better to make "ClusterCount" as const.

File: drivers/backup/portworx/portworx.go (hunk 1 above)
Comment (prashanthpx) [reply drivers/backup/portworx/portworx.go:751]: Fixed.

File: drivers/backup/portworx/portworx.go (hunk 2 above)
Comment (prashanthpx) [reply drivers/backup/portworx/portworx.go:750]: Fixed.

**Pull request merged**